marlowe-playground-client/package.json

```diff
@@ -17,15 +17,14 @@
   "dependencies": {
```

Comment:
Most of these changes were required to eliminate the `npm audit` warnings or even to allow `npm install` to work